doc/release_notes_0.2.rst

```diff
@@ -17,8 +17,63 @@ ACRN device model, and documentation.
 Version 0.2 new features
 ************************
 
+VT-x, VT-d
+================
+Based on Intel VT-x virtualization technology, ACRN emulates virtual CPU with core partition and simple schedule.
+VT-d stands for Intel VT for Directed IO, providing capabilities to assign I/O devices to VMs and extending the protection and isolation properties of VMs for I/O operations.
+
+PIC/IOAPIC/MSI/MSI-X/PCI/LAPIC
+================
+ACRN hypervisor supported APIC-V/EPT/IOAPIC/LAPIC functionality.
+
+Ethernet
+================
+ACRN hypervisor supported Ethernet functionality. Ethernet Mediator is executed in the Service OS and  provided packet forwarding between the physical networking devices (Ethernet, Wi-Fi, etc.) and virtual devices in the Guest VMs (also called "User OS"). Virtual Ethernet device could be shared by 1-2 Linux, Android, and Service OS guests for regular (i.e. non-AVB) traffic. All hypervisor para-virtualized I/O is implemented using the VirtIO specification Ethernet pass-through.
+
+Storage (eMMC)
+================
+ACRN hypervisor supported non-volatile R/W storage for the Service OS and Guest OS instances, supporting VM private storage and/or storage shared between Guest OS instances.
+
+USB (xDCI)
+================
+ACRN hypervisor supported assigning all USB xHCI and/or xDCI controllers to a Guest OS from the platform
+
+USB Mediator (xHCI and DRD)
+================
+ACRN hypervisor supported USB Mediator.
+
+CSME
+================
+ACRN hypervisor supported  CSME to a single Linux, Android, or RTOS guest or the Service OS even when in a virtualized environment.
+
+WiFi
+================
+ACRN hypervisor supported the passthrough assignment of the WiFi subsystem to the IVI, enabling control of the WiFI as an in-vehicle hotspot for 3rd party devices, providing 3rd party device applications access to the vehicle, and providing access of 3rd party devices to the TCU provided connectivity.
+
+IPU (MIPI-CS2, HDMI-in)
+================
+ACRN hypervisor supported passthrough IPU assignment to Service OS or guest OS, without sharing.
+
+Bluetooth
+================
+ACRN hypervisor supported bluetooth controller passthrough to single Guest OS (IVI).
+
+GPU  – Preemption
+================
+GPU Preemption is one typical automotive use case which requires the system to preempt GPU resources occupied by lower priority workloads in order to ensure performance of the most critical workload can be achieved. Three different schedulers for the GPU have to be involved: i915 UOS scheduler, Mediator GVT scheduler and i915 SOS scheduler.
```

Comment:
Change to:

    GPU Preemption is one typical automotive use case which requires
    the system to preempt GPU resources occupied by lower priority
    workloads.  This is done to ensure performance of the most critical
    workload can be achieved. Three different schedulers for the GPU
    are involved: i915 UOS scheduler, Mediator GVT scheduler, and
    i915 SOS scheduler.